relay-general/src/store/transactions.rs

```diff
@@ -395,16 +392,27 @@ mod tests {
             ..Default::default()
         });
 
-        assert_eq_dbg!(
-            process_value(
-                &mut event,
-                &mut TransactionsProcessor::new(None),
-                ProcessingState::root()
-            ),
-            Err(ProcessingAction::InvalidTransaction(
-                "trace context is missing op"
-            ))
-        );
+        process_value(
+            &mut event,
+            &mut TransactionsProcessor::new(None),
+            ProcessingState::root(),
+        )
+        .unwrap();
+        assert_annotated_snapshot!(event, @r###"
+        {
+          "type": "transaction",
+          "timestamp": 946684810,
+          "start_timestamp": 946684800,
+          "contexts": {
+            "trace": {
+              "trace_id": "4c79f60c11214eb38604f4ae0781bfb2",
+              "span_id": "fa90fdead5f74053",
+              "op": "op",
+              "type": "trace"
+            }
+          }
+        }
+        "###);
```

Comment:
Thanks Markus for showing me this macro.

`timestamp` and `start_timestamp` are set to match the values in the test, 2000-01-01 00:00:00, though I was still not able to determine why, when the tests run, those two fields become the current timestamp instead -- and the tests fail.